recipes/pymesh2/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.2.1" %}
+
+package:
+  name: pymesh2
+  version: {{ version }}
+
+source:
+  url: https://github.com/PyMesh/PyMesh/releases/download/v{{ version }}/pymesh2-{{ version }}-cp36-cp36m-linux_x86_64.whl
+  sha256: fd099114355b6793f584b387d6f18486564649902c252ff276638428c80bc358
+
+build:
+  number: 0
+  skip: True  # [not (linux and py36)]
+  script: "{{ PYTHON }} -m pip install --no-deps -vv --ignore-installed ./pymesh2-{{ version }}-cp36-cp36m-linux_x86_64.whl"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - scipy
+
+test:
+  imports:
+    - pymesh  # [not linux]
```

Comment:
skipped for linux?

Reply:
Yep. Just to fool the linter b/c it is GLIBC 2.14 and we don't have that. (The tests passes in a local installation of this artifact though.)

This is a study case b/c this package touches virtually everything we discuss here during the Package Summit, so fixing ti later is a way to show how we do things in conda-forge.